config/nightwatch.php

```diff
@@ -6,6 +6,7 @@
     'deployment' => env('NIGHTWATCH_DEPLOY'),
     'server' => env('NIGHTWATCH_SERVER', (string) gethostname()),
     'capture_exception_source_code' => env('NIGHTWATCH_CAPTURE_EXCEPTION_SOURCE_CODE', true),
+    'redact_headers' => explode(',', env('NIGHTWATCH_REDACT_HEADERS', 'Authorization,Cookie,Proxy-Authorization')),
```

Comment:
I kept this generic as figured if we captured outgoing request headers, it would always be the same list.

Alternatively, we could name this `redact_request_headers` and if we added `redact_outgoing_request_headers` in the future, we could always make it fallback to the same value.